samples/csharp/getting-started/MulticlassClassification_HeartDisease/HeartDiseaseConsoleApp/MulticlassClassification_HeartDisease.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <Content Include="..\Data\HeartTest.csv" Link="Data\HeartTest.csv" />
+    <Content Include="..\Data\HeartTraining.csv" Link="Data\HeartTraining.csv" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Data\" />
+    <Folder Include="Common\" />
+  </ItemGroup>
+
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="0.9.0" />
```

Comment:
Instead of:

`<PackageReference Include="Microsoft.ML" Version="0.9.0" />`

Change to:

`<PackageReference Include="Microsoft.ML" Version="$(MicrosoftMLVersion)" />`

So we get the version from a global location, file **Directory.Build.props** at /samples folder:

`<MicrosoftMLVersion>0.9.0</MicrosoftMLVersion>`

Internally this version is re-used for all the samples, except if overridden by any particular sample.